python/lsst/pipe/tasks/cli/cmd/opt/arguments.py

```diff
@@ -0,0 +1,47 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import click
+
+from lsst.daf.butler.cli.utils import (
+    MWArgumentDecorator,
+    split_commas,
+    unwrap,
+)
+
+
+num_subfilters_argument = MWArgumentDecorator(
+    "NUM-SUBFILTERS",
+    help="NUM-SUBFILTERS is the number of subfilters to be used for chromatic modeling.",
+    type=click.IntRange(min=0),
+    required=True
+)
+
+filter_names_argument = MWArgumentDecorator(
+    "FILTER-NAMES",
+    help=unwrap("""FILTER-NAMES names of the bands to define chromatic subfilters for in the registry. Each
+                band will have the same number of subfilters defined, for example 'g0', 'g1', and 'g2' for
+                three subfilters and band 'g'."""),
+    callback=split_commas,
+    required=True,
+    nargs=-1,
+)
```

Comment:
FWIW, I don't think there's any chance these arguments would be reused in any other butler or pipetask script, at least not without substantial changes, so if moving them into the file that uses them is consistent with how you'd like to organize things, feel free.

Reply:
So far I've kept the not-ad-hoc argument definitions in the `arguments.py` file. Even tho it's not likely to get reused it still seems more organized (and easy to search for later) than keeping it in `commands.py`.